src/main/java/org/usfirst/frc/team5687/robot/Constants.java

```diff
@@ -92,14 +102,19 @@
         public static final long PRIME_TIME = 1750;
 
         /**
+         * Time allowed for shooter wheels to stop before unpriming the boulder
+         */
+        public static final long UNPRIME_TIME = 1500;
+
+        /**
          * Speed to run the shooter wheels to shoot high goal
          */
         public static final double SHOOTER_SPEED = 1.0;
 
         /**
-         * Time allowed for shooter wheels to stop before unpriming the boulder
+         * Time allowed for shooter wheels to stop after firing to standby
          */
-        public static final long UNPRIME_TIME = 1500;
+        public static final long STANDBY_TIME = 1500;

```

Comment:
Wouldn't the standby and un-prime time be the same, since they both go from full-speed to stopped?